src/frontend/src/icp/types/env-token-ckerc20.ts

```diff
@@ -2,7 +2,8 @@ import { z } from 'zod';
 
 const envTokenData = z.object({
 	ledgerCanisterId: z.string(),
-	indexCanisterId: z.string()
+	indexCanisterId: z.string(),
+	erc20ContractAddress: z.string()
```

Comment:
I was curious to see if there was a zod way to check if the address starts with `0x` and landed on this SO answer https://ethereum.stackexchange.com/a/146766

WDYT?

In our setup:

```
import { isEthAddress } from '$lib/utils/account.utils';
import { z } from 'zod';

z.custom<string>(isEthAddress, "Invalid Address")
```

Reply:
i like it!